packages/labs/src/components/select/suggest.tsx

```diff
@@ -248,17 +248,26 @@ export class Suggest<T> extends React.Component<ISuggestProps<T>, ISuggestState<
 
     private handlePopoverWillClose = () => {
         const { popoverProps = {} } = this.props;
+        const { selectedItem } = this.state;
 
         // reset the query when the popover close, make sure that the list
         // isn't filtered on when the popover opens next
-        this.setState({ query: "" });
+        this.setState({
+            activeItem: selectedItem ? selectedItem : this.props.items[0],
```

Comment:
nit: Prefer explicit `selectedItem != null`. I like to avoid loose falsy checks.